Frontend/ui-driver/src/Screens/DriverEarningsScreen/View.purs

```diff
@@ -1917,7 +1917,7 @@ tagview name =
   imageView
     [ width (V 16)
     , height (V 16)
-    , imageWithFallback $ fetchImage FF_ASSET name
+    , imageWithFallback name
```

Comment:
name itself is a url?

Reply:
yes its url now

Reply:
did this because for one image we wanted `COMMON_ASSET` instead of `FF_ASSET`